src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpsParameters.java

```diff
@@ -24,9 +24,10 @@
  */
 
 package com.sun.net.httpserver;
+import javax.net.ssl.SSLParameters;
 import java.net.InetSocketAddress;
+
 //BEGIN_TIGER_EXCLUDE
```

Comment:
I don't have the context here - but if we're removing this import then I guess we should be removing the `// BEGIN` and `// END` comments too?

Reply:
Thanks for spotting this. IntelliJ re-arranged the imports automatically. I reverted it back to its original state

Reply:
I suspect the commented lines are obsolete TIGER was a long time ago.
That looks like some markup for a tool to use a non-ssl version as a demo. 
@AlanBateman Alan might know.